<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>394</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>PasswordChest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveFileAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAddPassword"/>
    <addaction name="actionEditPassword"/>
    <addaction name="actionDeletePassword"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewPasswords"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menu_Languages">
     <property name="title">
      <string>&amp;Languages</string>
     </property>
     <property name="icon">
      <iconset resource="../res/resource.qrc">
       <normaloff>:/icons/languages.png</normaloff>:/icons/languages.png</iconset>
     </property>
    </widget>
    <addaction name="menu_Languages"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Options"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSaveFile"/>
   <addaction name="separator"/>
   <addaction name="actionAddPassword"/>
   <addaction name="actionEditPassword"/>
   <addaction name="actionDeletePassword"/>
   <addaction name="separator"/>
   <addaction name="actionViewPasswords"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/open_file.png</normaloff>:/icons/open_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionAddPassword">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/add_password.png</normaloff>:/icons/add_password.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add password</string>
   </property>
  </action>
  <action name="actionEditPassword">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/edit_password.png</normaloff>:/icons/edit_password.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit password</string>
   </property>
  </action>
  <action name="actionDeletePassword">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/delete_password.png</normaloff>:/icons/delete_password.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete password</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/new_file.png</normaloff>:/icons/new_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionViewPasswords">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/view-password.png</normaloff>:/icons/view-password.png</iconset>
   </property>
   <property name="text">
    <string>View passwords</string>
   </property>
   <property name="toolTip">
    <string>View passwords</string>
   </property>
  </action>
  <action name="actionSaveFileAs">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/icons/save_as.png</normaloff>:/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSaveFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddPassword</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditPassword</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeletePassword</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionViewPasswords</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveFileAs</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAction()</slot>
 </slots>
</ui>
